generator/dart/generated/google_cloud_functions_v2/lib/cloudfunctions.dart

```diff
@@ -1320,7 +1320,7 @@ class SecretVolume extends Message {
 }
 
 /// Configuration for a single version.
-class SecretVolume$SecretVersion extends Message {
+class SecretVolume_SecretVersion extends Message {
```

Comment:
Same: Could this be called just `SecretVersion`? Or `SecretVolumeVersion`?
If there is no other `SecretVersion` in scope, then there is no conflict.
(Or even just have a `typedef SecretVersion = SecretVolume_SecretVersion;`.)

Do we fear that a later version will add another `SecretVersion` type, and we'll get a conflict at that time, and it'll be breaking to remove the shorthcut?

Reply:
More that I'd fear multiple enums in a library (all appropriately nested in messages), called `Type`, or `Status`, or something else very common.